x-pack/metricbeat/module/gcp/_meta/config.yml

```diff
@@ -69,3 +69,13 @@
   credentials_file_path: "your JSON credentials file path"
   dataset_id: "dataset id"
   table_pattern: "table pattern"
+
+ - module: gcp
+  metricsets:
+    - compute
+  region: "us-"
+  project_id: "your project id"
+  credentials_file_path: "your JSON credentials file path"
+  endpoint: http://your-endpoint
+  exclude_labels: false
+  period: 1m 
```

Comment:
The yaml block is not properly formatted and it might be redundant to add it again just to showcase the `endpoint` config param. I would probably add `endpoint` to an existing config in that file.